cadvisor.go

```diff
@@ -162,7 +163,11 @@ func main() {
 	}
 
 	containerLabelFunc := metrics.DefaultContainerLabels
-	if !*storeContainerLabels {
+
+	whiteListedLabels := strings.Split(*whiteListedContainerLabels, ",")
+	if len(whiteListedLabels) > 0 {
+		containerLabelFunc = metrics.WhiteListedContainerLabels(whiteListedLabels)
```

Comment:
See my comment about not separating `WhitelistedContainerLabels` and `BaseContainerLables`.  That simplifies this section as well, as you can just do a single assignment here.